doc/network-ng.md

```diff
@@ -61,7 +61,12 @@ The sysconfig backend support is composed by these files:
     ├── config_writer.rb <- WRITER
     ├── connection_config_reader.rb
     ├── connection_config_readers
+    │   ├── bonding.rb
+    │   ├── bridge.rb
+    │   ├── dummy.rb
     │   ├── ethernet.rb
+    │   ├── infiniband.rb
+    │   ├── vlan.rb
```

Comment:
I do not think we need to name all of them here. That `...` is enough to get idea that there is more types.